src/System.Diagnostics.DiagnosticSource/tests/System.Diagnostics.DiagnosticSource.Tests.builds

```diff
@@ -7,6 +7,10 @@
       <OSGroup>Windows_NT</OSGroup>
       <TestTFMs>netcore50;net46</TestTFMs>
     </Project>
+    <Project Include="System.Diagnostics.DiagnosticSource.Tests.csproj">
```

Comment:
Are .build files are no longer user for tests as well? I see many projects still have them. 
Can I assume tests will run for the targets provided in .csproj and safely remove .build file from the repo?